tests/x11/gnote/gnote_rename_title.pm

```diff
@@ -26,7 +26,12 @@ sub run {
     send_key "up";
     send_key "up";
     type_string "new title-opensuse\n";
-    $self->cleanup_gnote;
```

Comment:
We call same method in 3 more places, what exactly is the problem with it, because I have an impression that it was a good step to extract common code.

Reply:
In "lib/x11test.pm", it is waiting for "gnote-new-note-matched" tag. But it needs to match "gnote-new-note" on SLE. So, I have to revert this code.

Reply:
so now you doing exactly the same as in cleanup_gnote method with one tiny difference that you using another needle. What about refactoring cleanup_gnote to take needle as input and keep using it?